docs/development.md

```diff
@@ -1,128 +1,119 @@
+area exposed by the component (in other words, don't re-test imported
+`react-containers`). All styled views may also benefit from testing –
+asserting that various options affect one or two notable CSS values in
+question.
 
-### Views
+In cases where DOM structure or state would be unwieldy to determine,
+`data-test-*` attributes can be added in complex components to aid with
+testing. These attributes will be removed from published components during
+the build process.
 
-All view component styling must be based on an existing Garden
-designer-approved CSS component. Please see the [Garden CSS
-Components](https://github.com/zendeskgarden/css-components) repo for
-details.
+### Linting and formatting
 
-In addition all views must
+All JS/TS, CSS, and Markdown files are linted using eslint, stylelint, and
+markdownlint respectively. Additionally, prettier is used to format all
+JS/TS, and package.json files.
 
-- Be created using (or extending) a
-  [styled-components](https://www.styled-components.com/) primitive
-- Use the `retrieveComponentStyles` utility to allow dynamic theming
-- Provide the following analytics attributes:
-  - `data-garden-id="unique-component-id"` - this should match the ID
-    provided to `retrieveComponentStyles`
-  - `data-garden-version="packageVersion"` - provided by `package.json`
-  - (these attributes allow us to view current Garden usage across the
-    Zendesk product suite)
-- Provide `propTypes` for any custom visualizations you have provided
+## Package dependencies
 
-## Building Components
+Garden attempts to keep package dependencies to a minimum. Typical package
+dependencies include:
 
-The packages are built using webpack which creates several artifacts:
+- Dependencies
+  - `polished`
+- Peer dependencies
+  - `@zendeskgarden/react-theming`
+  - `prop-types`
+  - `react`
+  - `react-dom`
+  - `styled-components`
 
-- `styles.css`
-  - Each package has a single, minified, and scoped (css-modules) stylesheet
-    that includes all CSS consumed in the package
-- `commonjs` bundle
-  - Used by the majority of consumers
-  - All dependencies are `external` and not included in the bundle
-- `umd` bundle
-  - Can be consumed in browser with a standard `<script>` tag
-  - All dependencies are bundled to allow easy consumption
+When additional dependencies are required, they can be installed with the
+`lerna add` command (e.g. add `lodash.debounce` to the `test` package):
 
-## Testing Components
-
-All `Element` and `Container` components must only test the surface area that
-they are implementing. (i.e. Since `TabsContainer` is implemented with the
-`SelectionContainer` it does not need to test all keyboard navigation).
+```sh
+yarn lerna add lodash.debounce --scope @zendeskgarden/react-test
+```
 
-These tests should be implemented with standard assertions using the Enzyme
-shallow renderer when possible.
+Direct (non-peer) dependencies must be locked to the latest version (i.e. no
```

Comment:
> Direct (non-peer) dependencies must be locked to the latest version

Is this true for `dependencies`? We _want_ to have that open range to allow versions to be de-duped.

Reply:
Ah, meant to say "Dev dependencies". I'm just gonna drop the sentence because it is a mix that we check for on code review. i.e. https://github.com/zendeskgarden/react-components/blob/master/packages/tables/package.json